backend/azureblob/azureblob.go

```diff
@@ -88,6 +88,10 @@ Leave blank normally. Needed only if you want to use a service principal instead
 
 See ["Create an Azure service principal"](https://docs.microsoft.com/en-us/cli/azure/create-an-azure-service-principal-azure-cli) and ["Assign an Azure role for access to blob data"](https://docs.microsoft.com/en-us/azure/storage/common/storage-auth-aad-rbac-cli) pages for more details.
 `,
+		}, {
+			Name: "use_env_creds",
+			Help: `Pull credentials from AZURE_TENANT_ID and AZURE_CLIENT_{ID,SECRET} env vars a la EnvironmentCredential.`,
```

Comment:
What does "env vars a la EnvironmentCredential" mean? I'm not sure what `EnvironmentCredential` is - is that a bit of Azure jargon? I  found the reference - I think we should try to make that a bit clearer.

Note that the first line of this `Help` is used for the help text of the command line option, so I suggest you shorten the first line, then put more explanation in subsequent lines.

Maybe something like this (adapted from the s3 env_auth)

```go
Help: `Read credentials from runtime (environment variables).

Only applies if service_principal_file is blank. ???Maybe we should be checking the other auth too???

Pull credentials from AZURE_TENANT_ID and AZURE_CLIENT_{ID,SECRET} environment vars.

See EnvironmentCredential in the Azure docs for more info.
```

Reply:
Thanks for the suggestion; I've tweaked it a bit but hopefully in agreeable ways. :)